DAPIDLER_PROGRAM.s

```diff
@@ -154,7 +154,7 @@ CALLGMBL	EQUALS	BIT5		# RCSFLAGS INITIALIZATION.
 		TS	NEXTU
 		TS	NEXTV
 		CS	TEN
-		TS	DAPZRUPT	# JASK NOT IN PROGRESS, INITIALIZE NEG.
+		TS	DAPZRUPT	# TASK NOT IN PROGRESS, INITIALIZE NEG.
```

Comment:
`JASK` is present in the source code. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1412.jpg)

It's referenced in several places I've read, yet I still don't quite know what it actually is :confounded: